### PR TITLE
use Material App Bar in Page Header

### PR DESCRIPTION
**Pull request by britannio**
Closes https://github.com/WorldHealthOrganization/app/issues/996
Closes https://github.com/WorldHealthOrganization/app/issues/1249

IOS screen recording of all pages https://photos.app.goo.gl/rjMD61oAGFGGE1ue8
Android screen recording of all pages https://photos.app.goo.gl/uXKH6BGr4ZcXwrG56

- Switched to a Material App Bar for the `PageHeader` widget
- Removed all `CupertinoPageScaffold`s
- Matched the App Bar color with the color of the widget directly below it

#### How did you test the change?
* [ ] iOS Simulator
* [ ] Android Emulator
* [ ] iOS Device
* [x] Android Device
* [ ] `curl` to a dev App Engine server

---

#### Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).

- [x] REQUIRED: Do you have an Issue **assigned to you** for this PR?
- [x] Provided detailed pull request description and a succinct title
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

**Comment by davidkaneda** [comment]
Thanks @britannio, looking great. On both versions, the text seems a tad too far right when a back button is present. Did we add the extra padding to the title area vs. the overall app bar (when we tweaked alignment), or is that just default?

**Comment by britannio** [comment]
> Thanks @britannio, looking great. On both versions, the text seems a tad too far right when a back button is present. Did we add the extra padding to the title area vs. the overall app bar (when we tweaked alignment), or is that just default?

I added it to the text, I'll remove it when the back button is present.

**Comment by britannio** [comment]
image
image

@davidkaneda It's now using the default padding when the back button is present

**Comment by davidkaneda** [comment]
Awesome, thanks!

File: client/flutter/lib/pages/onboarding/legal_landing_page.dart [hunk 1]
```diff
@@ -15,84 +15,75 @@ class LegalLandingPage extends StatelessWidget {
...
-                          style:
-                              TextStyle(decoration: TextDecoration.underline),
-                          url: S
-                              .of(context)
-                              .legalLandingPagePrivacyPolicyLinkUrl,
-                          onLinkTap: (v) => launch(v),
-                        ),
-                        TextSpan(text: "."),
-                      ],
+                        color: Color(0xff008DC9),
```
Comment (hspinks) [inline client/flutter/lib/pages/onboarding/legal_landing_page.dart:34]:
```suggestion
                        color: Constants.primaryColor,
```

**Comment by davidkaneda** [comment]
Yeah, it's good to go-

**Comment by britannio** [comment]
@hspinks Can we get this in?

**Pull request merged**